Python/msrd_azure_upload.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env python3
+"""
+General Utility for managing Azure Powered File Uploads for MSRD.
+"""
+import json
+from pathlib import Path
+import datetime
+import os
+import click
+
+from azure.storage.blob import BlobPermissions, BlockBlobService
+
+def hard_exit(msg):
```

Comment:
Apply earlier comment about inlining and removing these functions.

Reply:
Done